src/main/kotlin/com/exactpro/th2/check1/rule/AbstractCheckTask.kt

```diff
@@ -76,7 +76,7 @@ abstract class AbstractCheckTask(
     private val ruleConfiguration: RuleConfiguration,
     submitTime: Instant,
     protected val sessionKey: SessionKey,
-    private val parentEventID: EventID,
+    protected val parentEventID: EventID,
     private val messageStream: Observable<StreamContainer>,
     private val eventBatchRouter: MessageRouter<EventBatch>
 ) : AbstractSessionObserver<MessageContainer>() {
```

Comment:
Please, take a look at `com.exactpro.th2.check1.rule.AbstractCheckTask#getCheckpointData`. This may be affected by changes in `grpc-common`

Reply:
Yes, thank you!